### DISCUSSION TITLE
Apollo federation V2 support ?

### DISCUSSION THREAD

**Discussion by paulrostorp** [graphql-hive/graphql-eslint#1066] [General]
Does this support apollo federation V2 ? 

I can't seem to get it to work when migrating, specifically when adding these new things to the schema: https://www.apollographql.com/docs/federation/federation-2/moving-to-federation-2/#opt-in-to-federation-2

**Comment by dimaMachina** [comment]
Please create a reproduction repo with your problem

**Comment by dimaMachina** [comment]
@paulrostorp If you have an error `Unknown directive @link` read my comment from https://github.com/B2o5T/graphql-eslint/discussions/1063#discussioncomment-2721160
